web/static/js/reducers/questionnaire.js

```diff
@@ -395,20 +440,27 @@ const validateSteps = (path, steps, context) => {
 }
 
 const validateStep = (path, step, context) => {
-  if (context.sms && isBlank(step.prompt['en'].sms)) {
+  if (context.sms &&
+      (!step.prompt[context.defaultLanguage] ||
+      !step.prompt[context.defaultLanguage].sms ||
+      isBlank(step.prompt[context.defaultLanguage].sms))) {
     addError(context, `${path}.prompt.sms`, 'SMS prompt must not be blank')
   }
 
-  if (context.ivr && step.prompt['en'].ivr && step.prompt['en'].ivr.audioSource == 'tts' && isBlank(step.prompt['en'].ivr.text)) {
+  if (context.ivr &&
+      step.prompt[context.defaultLanguage] &&
+      step.prompt[context.defaultLanguage].ivr &&
+      step.prompt[context.defaultLanguage].ivr.audioSource == 'tts' &&
+      isBlank(step.prompt[context.defaultLanguage].ivr.text)) {
     addError(context, `${path}.prompt.ivr.text`, 'Voice prompt must not be blank')
   }
 
-  if (step.type == 'multiple-choice') {
+  if (step.type === 'multiple-choice') {
     validateChoices(`${path}.choices`, step.choices, context)
   }
 }
 
-const validateChoices = (path, choices, context) => {
+const validateChoices = (path, choices: Choice[], context) => {
```

Comment:
This is assuming the `choices` structure of a regular Multiple Choice step. That's why I guard it from being called with Language Selection steps .